scriptoocloud/src/main/webapp/WEB-INF/views/password/change.html

```diff
@@ -0,0 +1,26 @@
+<html layout:decorate="~{layouts/main}">
+  <head>
+    <title>Authority Matcher</title>
+  </head>
+  <body>
+  <div layout:fragment="content">
+  <h2>Change password</h2>
+  <form action="#" th:action="@{/password/change}" method="POST" th:object="${user}">
+		<input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}" />
+		
+		<label>New Password:</label>
+		<input type="password" th:field="*{password}" class="form-control input-sm"></password>
```

Comment:
The key for this file is that it doesn't need th:object =${user} as the Professor said.  The real key is though, to change the th:field(s) to th:value instead.  If it sees th:field it expects a model object, I believe.